src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -255,108 +236,44 @@ protected override void ExtensionMethodChecks(BindingDiagnosticBag diagnostics)
             }
         }
 
-        protected override MethodSymbol FindExplicitlyImplementedMethod(BindingDiagnosticBag diagnostics)
-        {
-            var syntax = GetSyntax();
-            return this.FindExplicitlyImplementedMethod(isOperator: false, _explicitInterfaceType, syntax.Identifier.ValueText, syntax.ExplicitInterfaceSpecifier, diagnostics);
-        }
-
-        protected override Location ReturnTypeLocation => GetSyntax().ReturnType.Location;
-
-        protected override TypeSymbol ExplicitInterfaceType => _explicitInterfaceType;
```

Comment:
>protected override TypeSymbol ExplicitInterfaceType => _explicitInterfaceType; [](http://example.com/codeflow?start=8&length=78)

I think we should make this property abstract instead of removing it

Reply:
Unless it is abstract in the base.